contracts/SavingAccount.sol

```diff
@@ -74,6 +75,13 @@ contract SavingAccount is Initializable, InitializableReentrancyGuard, Constant,
         }
     }
 
+    function initFINAddress() public initializer {
```

Comment:
Also, we would be having a different address on another chain for FIN token. So I think we should have some `view` function called on FIN token in this function. This would ensure that the address is present on the chain. Otherwise, the deployment itself will fail.

Reply:
resolved

Reply:
How this is resolved ?